keps/sig-scheduling/785-scheduler-component-config-api/README.md

```diff
@@ -97,6 +97,14 @@ The second iteration, `kubescheduler.config.k8s.io/v1beta2`, includes the follow
     - `RequestedToCapacityRatio` (in favor of `NodeResourcesFit` plugin with a `RequestedToCapacityRatio` scoring strategy)
   - Cleanup of validation hacks.
 
+The third iteration, `kubescheduler.config.k8s.io/v1beta3`, includes the following changes:
```

Comment:
@ahg-g - You can update this once your KEP for simplified plugins merges.

Reply:
ack